java/ql/src/semmle/code/java/Expr.qll

```diff
@@ -1742,11 +1742,27 @@ class WildcardTypeAccess extends Expr, @wildcardtypeaccess {
   /** Gets the lower bound of this wildcard type access, if any. */
   Expr getLowerBound() { result.isNthChildOf(this, 1) }
 
+  /**
+   * Gets the bound of this wildcard type access, if any, i.e. either the upper
```

Comment:
Don't use abbreviations in qldoc.
```suggestion
   * Gets the bound of this wildcard type access, if any, that is, either the upper
```

Reply:
Made this a separate sentence ("... any. That is, ...") to be consistent with other comments and to not have so many commas.